iib/web/api_v1.py

```diff
@@ -458,7 +507,8 @@ def add_bundles():
     db.session.commit()
     messaging.send_message_for_state_change(request, new_batch_msg=True)
 
-    overwrite_from_index = payload.get('overwrite_from_index')
+    # TODO should not be there some check to None?
+    overwrite_from_index = bool(payload.get('overwrite_from_index'))
```

Comment:
Why typecast it? If a boolean is required, maybe we should default to `False`

Reply:
IIB docs say it should default to False, I just thought that there is a reason why this is not done.